keyboards/arabica37/keymaps/jp_default/keymap.c

```diff
@@ -0,0 +1,201 @@
+#include QMK_KEYBOARD_H
+
+extern uint8_t is_master;
+
+// Each layer gets a name for readability, which is then used in the keymap matrix below.
+// The underscores don't mean anything - you can have a layer called STUFF or any other name.
+// Layer names don't all need to be of the same length, obviously, and you can also skip them
+// entirely and just use numbers.
+enum layers {
+    _QWERTY,
+    _LOWER,
+    _RAISE,
+    _ADJUST,
+};
+
+enum custom_keycodes {
+  LOWER = SAFE_RANGE,
+  RAISE,
+  ADJUST,
+  RGBRST
+};
+
+
+#define KC______ KC_TRNS
+#define KC_XXXXX KC_NO
+#define KC_LOWER LOWER
+#define KC_RAISE RAISE
+#define KC_RST   RESET
+#define KC_LRST  RGBRST
+#define KC_LTOG  RGB_TOG
+#define KC_LHUI  RGB_HUI
+#define KC_LHUD  RGB_HUD
+#define KC_LSAI  RGB_SAI
+#define KC_LSAD  RGB_SAD
+#define KC_LVAI  RGB_VAI
+#define KC_LVAD  RGB_VAD
+#define KC_LMOD  RGB_MOD
+#define KC_CTLTB CTL_T(KC_TAB)
+#define KC_SFIME SFT_T(KC_ZKHK)
+#define KC_LOWDL LT(_LOWER, KC_DEL)
+#define KC_RAIBS LT(_RAISE, KC_BSPC)
+#define KC_ADJST MO(_ADJUST)
+#define KC_ALTHN ALT_T(KC_HENK)
+#define KC_GUIMH GUI_T(KC_MHEN)
+#define KC_SWAP  AG_SWAP
+#define KC_NORM  AG_NORM
+#define KC_DTR   SGUI(KC_RIGHT)
+#define KC_DTL   SGUI(KC_LEFT)
+#define KC_SCRS  LGUI(KC_PSCR)
+#define KC_RESET RESET
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+  [_QWERTY] = LAYOUT_kc( \
```

Comment:
Layout macros that automatically prepend `KC_` are discouraged, as they do not play well with the Configurator, and make some of the more advanced keycodes such as `MO()` and `LT()` harder to use, since you need to manually define aliases beginning with `KC_`.

Reply:
The default keymap doesn't use "KC_" macro, but I want to keep means of use it as another keymap. Should I still remove this?

Reply:
It's not THE `default` keymap, but it's still A default keymap.